spec.html

```diff
@@ -30619,6 +30628,7 @@ <h1>Properties of the Function Prototype Object</h1>
         <li>does not have a *"prototype"* property.</li>
         <li>has a *"length"* property whose value is *+0*<sub>𝔽</sub>.</li>
         <li>has a *"name"* property whose value is the empty String.</li>
+        <li>has a @@metadata property whose value is *null*, which is non-writable and non-configurable to prevent tampering that could be used to globally add metadata to all classes.</li>
```

Comment:
I think this should be its own section - we only really do "name" and "length" like this.

Compare, for example, [`GeneratorFunction.prototype [ @@toStringTag ]`](https://tc39.es/ecma262/#sec-generatorfunction.prototype-@@tostringtag). (Except of course it should be non-configurable, and then it can be just "the value" rather than "the initial value".)

I'm neutral on whether to include the explanation in that section as well or just leave it out entirely.

Reply:
Ah, I was looking for something similar and couldn't find it, I'll update!